src/katsdpnetboxutilities/system/device-md.py

```diff
@@ -278,17 +325,25 @@ def _add_memory(self):
         self.page.heading("Memory", 2)
         core = self._device_info.lshw_core()
         for child in core.get("children", []):
-            if child.get("id") in ["memory", "memory:0", "memory:1","memory:2", "memory:3", "memory:4", "memory:5",
-                                   "memory:6"]:
-            #if "memory" in child.get("id"):
+            if child.get("id") in [
```

Comment:
This just looks like a bug...... static enumeration. We need to look at the source docs and see if this will always work.